integration-tests/test-extension/extension-that-defines-junit-test-extensions/deployment/src/main/resources/application.properties

```diff
@@ -1,193 +0,0 @@
-# Log settings
-# log level in lower case for testing
-quarkus.log.level=info
```

Comment:
Why is this file removed? It was useless?

Reply:
When I initially wrote these tests, I'd copied the files from another test. It had a whole bunch of settings that we didn't actually want in this test, such as setting the non-application-root to 1234. (Although, as mentioned above, even without the settings file, the non-application root is *still* 1234, which I haven't managed to explain.)

I left the file, so we had change history, but removed all of the settings since we didn't need any of them, as far as I know.